### PR TITLE
Update Portability Service Gateway routes

### PR DESCRIPTION
**Pull request by mjrousos**

### CODE REVIEW

File: deploy/PortabilityService.ServiceFabricApplication/DeploySF.ps1 [hunk 1]
```diff
@@ -131,6 +137,10 @@ if ($EnvironmentName)
 {
     $parameters.EnvironmentName = $EnvironmentName
 }
+if ($ImageTag)
+{
+    $parameters.ImageTag = $ImageTag
```
Comment (twsouthwick) [inline deploy/PortabilityService.ServiceFabricApplication/DeploySF.ps1]: What's the point of this if the tags cannot be parameterized?

Comment (mjrousos) [reply deploy/PortabilityService.ServiceFabricApplication/DeploySF.ps1]: Just added for future use. Though researching, there may not be an easy way to use it so I'll remove it for now.

**Comment by mjrousos** [comment]
SF doesn't allow the tags to be parameters so we'll have to do some token substitution ourselves.

**Comment by conniey** [comment]
@mjrousos Is this ready to be merged?

**Comment by twsouthwick** [comment]
Should we file a feature request to allow that to be parametized? It would make deployment much easier

**Pull request merged**